net/youtube-dl/Portfile

```diff
@@ -37,15 +37,20 @@ if {${subport} eq ${name}} {
 }
 
 subport yt-dlp {
-    github.setup    yt-dlp ${subport} 2025.10.22
+    github.setup    yt-dlp ${subport} 2025.11.12
     revision        0
-    checksums       rmd160  83ddc3b5c6fa99da2554ba48039d8d82d934e2f3 \
-                    sha256  8ed2971172cbe1295258341e9f04527a7f27578538ce70ef4c9ebaba8966a969 \
-                    size    6025669
+    checksums       rmd160  b204b191aa2c5abd0bb2f66a1e75d70b112837b7 \
+                    sha256  561ead2e2991f11789d3cd9f9b5a2eee2641a587e2cfc1ee9631c7c1d851c0ec \
+                    size    6139146
     dist_subdir     ${subport}/${version}
     distname        ${subport}
 
     python.pep517_backend hatch
+    depends_run-append  port:yt-dlp-ejs
+
+    variant no_javascript description {Remove yt-dlp-ejs and javascript runtime dependency} {
```

Comment:
Any reason to make it a negative variant instead of positive, but default one?

Reply:
adding a variant for something that seems likely to be a fundamental part of of how yt-dlp works in the future (ie., solving JS challenges to download media) seemed more like clutter than anything else. But i'll defer to @ryandesign.